notebooks/official/feature_store/vertex_ai_feature_store_feature_view_service_agents.ipynb

```diff
@@ -0,0 +1,840 @@
+{
```

Comment:
<p>Use&nbsp;<a href="https://developers.google.com/style/person" target="_blank">second person</a>&nbsp;and an imperative style - "First, create a FeatureGroup"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2752/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Fixed.</p>